### PR TITLE
docs: usage for staging bucket permissions

### PR DESCRIPTION
**Pull request by billgeo**
I had trouble setting up permissions from my test bucket, so thought it would be good to have some helpful pointers for this. I ended up using this policy, although not sure it is the best/safest example?

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -1,16 +1,38 @@
...
+        {
+            "Sid": "data-lake-readonly",
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::<DATALAKE_AWS_ACCOUNT_ID>:root"
+            },
+            "Action": [
+                "s3:GetObject"
+            ],
+            "Resource": "arn:aws:s3:::<YOUR_STAGING_BUCKET>/<YOUR_DATASET>"
```
Comment (imincik) [inline USAGE.md:30]: I think it should be `Resource": "arn:aws:s3:::<YOUR_STAGING_BUCKET>/<YOUR_DATASET>/*` (adding slash and asterisk at the end)

Comment (billgeo) [reply USAGE.md]: Thanks. Fixed that.

**Comment by billgeo** [comment]
@imincik, please re-review.

**Review by imincik — APPROVED** [review]
Thanks @billgeo

**Pull request merged**